lib/raven/configuration.rb

```diff
@@ -169,6 +169,11 @@ class Configuration
     # }
     attr_reader :before_send
 
+    # Job queues are retried when an exception is raised.
+    # Setting this option to true will wait untill all retries have passed before
+    # reporting the exception to Sentry
+    attr_accessor :report_after_job_retries
```

Comment:
I think the following is clearer: Set this option to true if you want Sentry to only capture the last job retry if it fails.